modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1227,6 +1228,47 @@ Suggestions for improvement::
 Use `DATABASE *` without the parameter to revoke the privilege on all databases.
 ====
 
+[#_topology_notifications]
+== `TOPOLOGY` category
+
+Topology notifications indicate that the query or command connected to managing
+databases and/or servers has some additional information to tell the user.
+
+[#_neo_clientnotification_cluster_unnecessaryenableserver]
+=== UnnecessaryEnableServer
+
+.Notification details
+[cols="<1s,<4"]
+|===
+|Code
+m|Neo.ClientNotification.Cluster.UnnecessaryEnableServer
+|Title
+a|Provided server already enabled
+|Severity
+m|INFORMATION
+|Category
+m|TOPOLOGY
+|===
+
+.Enabling an already enabled server
+====
+Command::
++
+[source, cypher]
+----
+ENABLE SERVER "123e4567-e89b-12d3-a456-426614174000"
+----
+
+Full title of the returned code::
```

Comment:
I am a bit confused. Why do we need a title and a full title? Currently, notifications have a title and a description of the returned code. I think these lines should compose the description.

Reply:
Yes in 5.x there is one title and one description, while in 6.x there will be a message so we should think this split through a bit more